src/ml_board/frontend/Dashboard/src/worker_socket/MainThreadCallbacks.ts

```diff
@@ -6,7 +6,7 @@ import handleJobStatusData from "./event_handlers/JobStatusHandler";
 
 // Note: I don't like having anything here other than the callbacks but it is what it is :')
 // ========================= variables ============================//
-// TODO: in #145 #147 this has been done with out the need of this, check if still needed (check the TODO drafts)
+// TODO: in #145 #147 this has been done with out the need of this, check if still needed (check #150)
 // to check for metric and loss headers
 const metric_loss_header: string[] = [];
 // TODO: temporary until the mystery why this is even used be solved
```

Comment:
if this is not used anywhere, then it can be also removed. Might be legacy code that has not been refactored out, yet?